authors/apps/articles/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.1.2 on 2018-11-06 09:25
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import taggit.managers
+
+
+class Migration(migrations.Migration):
```

Comment:
I think it would be best not to push the migrations to GitHub and ignore the entire migrations file as earlier agreed.

Please look into this

Reply:
I had issues in on my local machine, so I had to delete the database, so to run migrations again I had to delete the migrations folder and run them when I pushed to GitHub, Travis started failing complaining about not seeing some migrations file, which made me push the folder. This won't happen again.